examples/DictionaryExample/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Momento.Sdk.Exceptions;
+using Momento.Sdk.Incubating;
+
+public class Driver
+{
+    private static readonly string AUTH_TOKEN_ENV_VAR = "TEST_AUTH_TOKEN";
+    private static readonly string CACHE_NAME_ENV_VAR = "TEST_CACHE_NAME";
+
+    public async static Task Main()
+    {
+        var authToken = ReadAuthToken();
+        var cacheName = ReadCacheName();
+
+        using var client = Momento.Sdk.Incubating.SimpleCacheClientFactory.CreateClient(authToken, 60);
+        EnsureCacheExists(client, cacheName);
+
+        // Set a value
+        await client.DictionarySetAsync(cacheName: cacheName, dictionaryName: "my-dictionary",
+            field: "my-field", value: "my-value", refreshTtl: false, ttlSeconds: 60);
+
+        // Set multiple values
+        await client.DictionarySetBatchAsync(
+            cacheName: cacheName,
+            dictionaryName: "my-dictionary",
+            new Dictionary<string, string>() {
+                { "field1", "value1" },
+                { "field2", "value2" },
+                { "field3", "value3" }},
+            refreshTtl: false);
+
+        // Get a value
+        var field = "field1";
+        var getResponse = await client.DictionaryGetAsync(
+            cacheName: cacheName,
+            dictionaryName: "my-dictionary",
+            field: field);
+        var status = getResponse.Status; // HIT
+        string value = getResponse.String()!; // "value1"
+        Console.WriteLine($"Dictionary get of {field}: status={status}; value={value}");
+
+        // Get multiple values
+        var batchFieldList = new string[] { "field1", "field2", "field3", "field4" };
+        var getBatchResponse = await client.DictionaryGetBatchAsync(
+            cacheName: cacheName,
+            dictionaryName: "my-dictionary",
+            fields: batchFieldList);
+        var manyStatus = getBatchResponse.Status; // [HIT, HIT, HIT, MISS]
+        var values = getBatchResponse.Strings(); // ["value1", "value2", "value3", null]
+
+        Console.WriteLine("\nDisplaying the result of dictionary get batch:");
+        foreach ((var batchField, var response) in batchFieldList.Zip(getBatchResponse.Responses))
```

Comment:
Discussion point not intended to block this PR, but before we finalize these APIs... is a Zip really how we want customers to have to deal with this?  As opposed to returning an actual Map/Dictionary object?

Reply:
I used zip to just collect the keys alongside the responses, since the responses contain only the values

Reply:
yeah i get it now.  When i Initially reviewed this I thought you were soliciting API feedback as opposed to just feedback on an example written against the current APIs.

I'd definitely be interested in discussing some final details on this particular API before we lock it down, but now that I understand your intent for this PR I think that is an orthogonal conversation.